app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,11 +1,331 @@
+      <h4>Delivery</h4>
+
+      <p>When the truck arrives, you will do a walk-through with the driver and the delivery crew. Identify any pre-existing damage to the residence and let them see the layout of the home to determine the best way to deliver your household goods.</p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Number your rooms, so that if there is a language barrier in your new host country, everyone will know where the boxes should go.
+      <% end %>
+
+      <p>The delivery crew should put down some type of material (plastic, fabric, etc.) to protect some of your flooring and expect them to put some form of protection on any tight corners that may be in the residence as well.</p>
+
+      <p>Get an <strong>inventory sheet</strong> from the driver so you can check off the numbers and ensure that everything is delivered. Doing this yourself is in your best interest. As the boxes come off the truck, you should expect to direct them to a room in the residence where that item should be placed.</p>
+
+      <div class="usa-grid-full">
+        <div class="usa-width-two-thirds">
+          <p>Once the truck is empty, verify on the inventory sheet that everything has been delivered. If not, those items need to be annotated on the inventory sheet before you sign off. Additionally, if you see any items come in damaged you can annotate that as well, but you’ll still have time to file a claim in case you missed it. This goes for any damage that may have occurred to the residence.</p>
+        </div>
+        <div class="usa-width-one-third">
+          <%= render layout: 'shared/pro_tip' do %>
+            <strong>Pro-Tip:</strong> Enlist your family or your Sponsor for help during this during this phase.
+          <% end %>
+        </div>
+      </div>
+
+      <p>Finally, you have a couple of options during the delivery phase about what items you want the delivery crew to unpack.</p>
+
+      <ul>
+	      <li>A full unpack: Movers empty the boxes of all their contents into the specified rooms.</li>
+	      <li>A partial unpack: Movers only open boxes in select areas.</li>
+        <li>Movers reassemble furniture that they took apart at the origin (beds, couches, etc.)</li>
+      </ul>
+    </div>
+
+    <div id="clean-up" class="article-section">
+      <h4>Clean Up, Claims &amp; Customer Survey</h4>
+
+      <p>
+        <strong><em>Clean-Up</em></strong><br>
+        Once you have emptied out all your household goods from the boxes, call the local moving company and set up a time for them to come back out and remove any old boxes, packing material, etc. They are responsible for removing it from the residence one time after delivery.
+      </p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Save time by piling up all the boxes and moving supplies in one location!
+      <% end %>
+
+      <p>
+        <strong><em>Claims</em></strong><br>
```

Comment:
Is the content in this section identical to that on other pages? If so, should it be a partial?

Reply:
The content is nearly identical and probably should be identical. I'll make a partial